knowledge-base/kb-changehandicon-to-loadingicon.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Change Hand Icon to Any Loading Icon Upon Clicking on Filter
+description: "How to change hand icon to any loading icon upon clicking on filter"
+type: how-to
+page_title: How to change hand icon to any loading icon upon clicking on filter for WPF 
```

Comment:
Each word in the "title" and "page_title" properties should begin with an uppercase letter.